Makefile

```diff
@@ -14,7 +14,7 @@ help:
 
 EXTRA_BUILD_ARGS?=
 .PHONY: build
-build: ## Builds the project
+build: proto add-license ## Builds the project
```

Comment:
Instead of having `add-license` as a dependency here we could instead append `$(MAKE) add-license` to the commands in the `proto` target. This would ensure that the license headers are not removed when running `make proto` on its own.

Reply:
That's a good point. After thinking about it a bit more I would rather stick to the current approach.

I am looking at those two targets as separate, smaller units of work which you could "pipe".

Unless there's a way to supply a header (which I haven't looked at yet) similarly to what `controller-gen` exposes through `object:headerFile` I would prefer to stick to this composition. 

Especially if we add some more dependent targets to the `build` which might need adding license as well.